src/util/usePanelParams.tsx

```diff
@@ -73,6 +78,9 @@ const usePanelParams = (): PanelHelper => {
     newParams.delete("profile");
     newParams.delete("panel-project");
     newParams.delete("sub-form");
+    newParams.delete("bucket");
+    newParams.delete("bucketpool");
```

Comment:
```suggestion
    newParams.delete("panel-pool");
```
I think we renamed this below.